spec.bs

```diff
@@ -1045,7 +1141,66 @@ Note: Previously, the `payment` extension allowed for the creation of
               enrollment.https.html
             </wpt>
 
-:  <dfn export>Client extension processing ([=authentication extension|authentication=])</dfn>
+    1. Before step 13 (creating a {{CollectedClientData}}).
+
+        1. Let |bbk_allowed_algorithms| be {{AuthenticationExtensionsPaymentInputs/browserBoundPubKeyCredParams}}.
+
+        1. Let (|bbk| - the platform specific keypair handle, |bbk_id| - an
+            identifier of the |bbk|) be the result of
+            {{BrowserBoundKeyStore/CreateKeyPair}}(|bbk_allowed_algorithms|).
+
+            <dl class="switch">
+
+                : BBK keypair was not created
+                :: Skip steps below relating to |bbk| and |bbk_public_key|.
+
+            </dl>
+
+        1. Let |bbk_public_key| be the result of
+            {{BrowserBoundKeyStore/GetBrowserBoundPublicKey}}(|bbk|).
+
+    1. In step 13, instead of creating the {{CollectedClientData}} create a {{CollectedClientPaymentData}} whose fields are:
+
+        1. {{CollectedClientPaymentData/payment}} set to a {{CollectedClientAdditionalPaymentData}} with
```

Comment:
Ah, this is tricky. `CollectedClientAdditionalPaymentData` has a bunch of `required` members currently, and so you cannot construct one with just the `browserBoundPublicKey` field set. I don't have an answer for this immediately, we'll have to figure out what we want to do here (include the other members? make them optional? - technically backwards incompatible but only very minorly given this is basically an internal data structure. or define a different CollectedClientAdditionalPaymentData for registration time?)